package.json

```diff
@@ -35,72 +38,68 @@
   "author": "Andrey Okonetchnikov @okonetchnikov",
   "contributors": [
     "Andrey Okonetchnikov <andrey@okonet.ru> (http://okonet.ru)",
-    "Roland Groza <rolandjitsu@gmail.com>"
+    "Roland Groza <rolandjitsu@gmail.com>",
+    "Rick Markins <rmarkins@gmail.com>"
+  ],
+  "files": [
+    "dist",
+    "src",
+    "README.md",
+    "index.d.ts"
   ],
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/react-dropzone/attr-accept/issues"
   },
   "homepage": "https://github.com/react-dropzone/attr-accept#readme",
   "devDependencies": {
-    "@babel/cli": "^7.6.4",
-    "@babel/core": "^7.10.5",
-    "@babel/plugin-external-helpers": "^7.2.0",
-    "@babel/plugin-proposal-do-expressions": "^7.6.0",
-    "@babel/plugin-proposal-export-default-from": "^7.5.2",
-    "@babel/plugin-proposal-logical-assignment-operators": "^7.2.0",
-    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.4.4",
-    "@babel/plugin-proposal-optional-chaining": "^7.6.0",
-    "@babel/plugin-proposal-pipeline-operator": "^7.5.0",
-    "@babel/plugin-transform-runtime": "^7.6.2",
-    "@babel/preset-env": "^7.6.3",
-    "@babel/preset-react": "^7.6.3",
-    "@babel/register": "^7.6.2",
-    "@commitlint/cli": "^8.2.0",
-    "@size-limit/preset-small-lib": "^2.1.6",
-    "babel-eslint": "^8.0.0",
-    "babel-loader": "^8.0.0",
-    "babel-preset-env": "^1.6.0",
-    "core-js": "^3.1.4",
-    "cross-env": "^6.0.3",
-    "cz-conventional-changelog": "^1.2.0",
-    "eslint": "^4.6.1",
-    "eslint-config-airbnb": "^15.1.0",
-    "eslint-config-okonet": "^6.1.3",
-    "eslint-config-prettier": "^2.4.0",
-    "eslint-plugin-flowtype": "^2.35.1",
-    "eslint-plugin-import": "^2.7.0",
-    "eslint-plugin-jsx-a11y": "^5.1.1",
-    "eslint-plugin-node": "^5.1.1",
-    "eslint-plugin-prettier": "^2.2.0",
-    "eslint-plugin-react": "^7.3.0",
-    "expect": "^1.20.2",
-    "husky": "^0.14.3",
-    "lint-staged": "^4.1.3",
-    "mocha": "^3.5.0",
-    "prettier": "^1.6.1",
-    "rimraf": "^3.0.0",
-    "webpack": "^4.0.0",
-    "webpack-cli": "^3.3.12"
+    "@babel/core": "^7.25.8",
+    "@commitlint/cli": "^19.5.0",
+    "@eslint/js": "^9.12.0",
+    "@size-limit/preset-small-lib": "^11.1.6",
+    "browserslist": "^4.24.0",
+    "cz-conventional-changelog": "^3.3.0",
+    "eslint": "^9.12.0",
+    "eslint-plugin-import": "^2.31.0",
+    "expect": "^29.7.0",
+    "husky": "^9.1.6",
+    "lint-staged": "^15.2.10",
+    "mocha": "^10.7.3",
+    "parcel": "^2.12.0",
+    "rimraf": "^6.0.1",
+    "typescript": "^5.6.3"
   },
   "engines": {
-    "node": ">=4"
+    "node": ">=18"
```

Comment:
If we require node 18+, then I don't see why we should keep the commonjs support. Beyond this version, ESM is fully supported and there's no need to transpile. So do most bundlers as far as I can tell. Similarly, browser support for ESM is marked as baseline, so  it's widely supported.